config/routes.rb

```diff
@@ -16,6 +16,10 @@
   end
 
   scope module: "dashboard" do
+    get 'batches/new', to: 'batches#new', as: 'new_batch'
+    post 'batches/', to: 'batches#create', as: 'batches'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.